docs/src/api/params.md

```diff
@@ -247,6 +247,37 @@ The file path to save the storage state to. If [`option: path`] is a relative pa
 current working directory. If no path is provided, storage
 state is still returned, but won't be saved to the disk.
 
+## js-python-context-option-har
+* langs: js, python
+- `har` <[Object]>
+  - `path` <[path]> Path to a [HAR](http://www.softwareishard.com/blog/har-12-spec) file with prerecorded network data. If the HAR file contains an entry with the matching URL and HTTP method, then the entry's headers, status and body will be used to fulfill the network request. An entry resulting in a redirect will be followed automatically. If there is no matching entry in the file the execution continues to try other configured HAR files and [Route] handlers. If `path` is a relative path, then it is resolved relative to the current working directory.
+  - `fallback` ?<[HarFallback]<"abort"|"continue">> If set to 'abort' any request not found in the HAR file will be aborted. If set to'continue' missing requests will continue normal flow and can be handled by [Route] handlers. Defaults to 'abort'.
```

Comment:
"can be handled by [Route] handlers" is not true, since HAR is the last route in the chain. Let's say it will go to the actual network.